helm-chart/sefaria-project/values.yaml

```diff
@@ -105,7 +105,7 @@ web:
       ref:
 
 linker:
-  enabled: false
```

Comment:
The intention is not to enable the linker as default behaviour, which is what this does.  The production specific values are in build/ci/production-values.yaml. I need to add a couple extra lines to the deploy script to populate the deploy tag for the linker too, I will add enabling the linker to that change

Reply:
Got it. Sorry for the mix-up. We still need the change to the linker image though below, right? that should be default for all usages of linker.